### PR TITLE
virt: add a parameter for the autostart command

### PR DESCRIPTION
**Pull request by debugloop**
This commit adds a simple boolean switch to determine wether the
autostart flag of a domain should be set or not. Previously, the only
option was to set it. This commit preserves this default, so no change
should be noticable.

I don't think there is an issue associated with this yet.

### CODE REVIEW

**Comment by robynbergeron** [comment]
Thanks @analogbyte for this PR. This module is maintained by the Ansible core team, so it can take a while for patches to be reviewed. Thanks for your patience.

File: cloud/misc/virt.py [hunk 1]
```diff
@@ -55,6 +55,13 @@
       - XML document used with the define command
     required: false
     default: null
+  autostart_state:
+    description:
+      - Boolean switch which determines the action taken by the autostart
+        command. 'True' indicates a guest should autostart, 'False' the
+        opposite.
+    required: false
+    default: "True"
```
Comment (mscherer) [inline cloud/misc/virt.py:64]: You need to add version_added: "2.1" here.

Comment (debugloop) [inline cloud/misc/virt.py:64]: Done. Thanks for the feedback, I'll think of it next time.

**Comment by gregdek** [comment]
@analogbyte A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes.

**Comment by debugloop** [comment]
Thanks for the heads up, I thought adding the requested change was enough to show my intentions :)
But yes, this is still valid, having the autostart flag and being unable to unset it is not what people want from a module managing libvirt.

**Comment by gregdek** [comment]
Thanks @analogbyte for this PR. This PR requires revisions, either because it fails to build or by reviewer request. Please make the suggested revisions. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by debugloop** [comment]
ready_for_review

**Comment by gregdek** [comment]
Thanks @analogbyte for this PR. This module is maintained by the Ansible core team, so it can take a while for patches to be reviewed. Thanks for your patience.

Core team: please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with 'needs_revision' or merge as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

**Pull request closed without merging**